src/server/graphql-server.ts

```diff
@@ -57,12 +57,12 @@ export class PolarisGraphQLServer implements GraphQLServer {
                         body: context.request.body,
                     };
                 } catch (e) {
-                    this.polarisLogger.error('headers error', { throwable: e });
+                    this.polarisLogger.error('Headers error', { throwable: e });
```

Comment:
write a more descriptive message
and what about tests for getHeaders() method?
does it fails if someone sends an header that is not listed in the headers schema (bad)? 
does it fails if someone does not send an header that is listed in the headers schema (bad)?

Reply:
the descriptive message will appear in the throwable.
getHeaders method will be tested separately.